samples/boards/argonkey/README.rst

```diff
@@ -0,0 +1,53 @@
+.. _ArgonKey:
+
+ArgonKey sensor board
+#####################
+
+Overview
+********
+This sample provides an example of how to read sensor data
+from the ArgonKey board. The result is displayed on the console.
+It makes use of both the trigger and poll methods.
+
+Requirements
+************
+
+This sample just requires the ArgonKey board. The user may select
+or unselect the sensors from prj.conf.
```

Comment:
Change to say:

     ``samples/boards/argonkey/prj.conf``

If I unselect one of the sensors, will the build fail or just not output meaningful data for that sample because the sensor is disabled?

Reply:
Everything is under ifdefs, so it will build fine and just avoid to configure/print sensor drivers not present.